### PR TITLE
dx-808 Restructured into steps.

### PR DESCRIPTION
**Pull request by arebra**
REstructured the seamless view section for Invoice

### CODE REVIEW

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 24
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5231986&bid=17152132)

File: payments/invoice/seamless-view.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ sidebar:
       title: Redirect
     - url: /payments/invoice/seamless-view
       title: Seamless View
+    - url: /payments/card/capture
+      title: Capture
```
Comment (payex-dev) [inline payments/invoice/seamless-view.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5231986)

File: payments/invoice/seamless-view.md [hunk 2]
```diff
@@ -10,6 +10,8 @@ sidebar:
       title: Redirect
     - url: /payments/invoice/seamless-view
       title: Seamless View
+    - url: /payments/card/capture
```
Comment (payex-dev) [inline payments/invoice/seamless-view.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5231986)

Comment (NullableInt) [reply payments/invoice/seamless-view.md]: Wrong

File: payments/invoice/seamless-view.md (hunk 1 above)
Comment (NullableInt) [reply payments/invoice/seamless-view.md]: Wrong

Comment (asbjornu) [reply payments/invoice/seamless-view.md]: Indeed

File: payments/invoice/seamless-view.md (hunk 2 above)
Comment (asbjornu) [reply payments/invoice/seamless-view.md]: Wrong indeed

File: payments/invoice/seamless-view.md [hunk 3]
```diff
@@ -32,114 +34,9 @@ does not need to leave your webpage, since we are handling the payment in the
...
-Note that the invoice will not be created/distributed before you have
-made a `capture` request." %}
-
-The Capture/Cancel/Reversal options are described in the [after payment
-section][after-payment]. The link will take you directly to the API descriptions
-for the requests.
-
-### Explainations
-
-* ① Start with collecting all purchase information and make a `POST` request
```
Comment (asbjornu) [inline payments/invoice/seamless-view.md]: Shouldn't we have this list below the diagram?

Comment (arebra) [reply payments/invoice/seamless-view.md]: Sure, I can put it back in :) Only the explanations or the API requests as well?

Comment (asbjornu) [reply payments/invoice/seamless-view.md]: Only the list

**Pull request merged**